File: app/src/main/java/ep/appobject/MainActivity.java
package ep.appobject;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;

public class MainActivity extends AppCompatActivity {

    private EditText name;
    private Button buttonSave, buttonHelp;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        name = findViewById(R.id.name);
        buttonSave = findViewById(R.id.button_save);
        buttonSave.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                saveToAppObject();
            }
        });

        buttonHelp = findViewById(R.id.button_help);
        buttonHelp.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                goToHelpActivity();
            }
        });
    }

    private void goToHelpActivity() {
        startActivity(new Intent(this, HelpActivity.class));
    }

    // TODO shranite ime v objekt aplikacije
    public void saveToAppObject() {

    }
}

File: app/src/main/java/ep/appobject/MainActivity.kt
package ep.appobject

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import ep.appobject.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.buttonSave.setOnClickListener { saveToAppObject() }
        binding.buttonHelp.setOnClickListener {
            startActivity(Intent(this, HelpActivity::class.java))
        }
    }

    // TODO shranite ime v objekt aplikacije
    private fun saveToAppObject(): Unit = TODO()
}